src/encoded/viewconfigs/matrix.py

```diff
@@ -202,50 +197,68 @@ def preprocess_view(self):
                 type_info = self._types[self._doc_types[0]]
                 self._schema = type_info.schema
         self._validate_items(type_info)
-        self._result['title'] = self._set_result_title(type_info)
-        self._result['matrix'] = type_info.factory.matrix.copy()
-        self._matrix = self._result['matrix']
-        self._matrix['x']['limit'] = self._request.params.get('x.limit', 20)
-        self._matrix['y']['limit'] = self._request.params.get('y.limit', 5)
+        matrix = type_info.factory.matrix.copy()
+        matrix['x']['limit'] = self._request.params.get('x.limit', 20)
+        matrix['y']['limit'] = self._request.params.get('y.limit', 5)
         search_route = self._request.route_path('search', slash='/')
-        self._matrix['search_base'] = search_route + self._search_base
-        matrix_route = self._request.route_path('matrix', slash='/')
-        self._matrix['clear_matrix'] = matrix_route + '?type=' + self._doc_types[0]
-        self._result['views'] = self._construct_result_views(type_info)
-        query, used_filters = self._construct_query()
-        es_results = self._elastic_search.search(body=query, index=self._es_index)
+        matrix['search_base'] = search_route + self._search_base
+        matrix['clear_matrix'] = matrix_route + '?type=' + self._doc_types[0]
+        matrix_x_y = {
+            'x': matrix['x'],
+            'y': matrix['y'],
+        }
+        search_query, used_filters = self._construct_query(
+            result_filters,
+            matrix_x_y
```

Comment:
Yes, you are correct and I normally try to keep name consistent. I used the name "matrix_x_y" in process_view to avoid a name collision in this method with "matrix". However, for methods called by process_view directly or indirectly, like "construct_query", they properly should use "matrix". This also applies to other use of "matrix" in other classes.Do you prefer I used "matrix_x_y" anyway?

Reply:
Which other classes use these MatrixView functions?